cmdget.go

```diff
@@ -99,10 +99,14 @@ func runGet1(args []string) error {
 			}
 			repl = repl1
 		}
-		// Automatically generate a go.mod file if one doesn't
-		// already exist, because otherwise the directory cannot be
-		// used as a module.
-		if err := ensureGoModFile(repl.modulePath, repl.dir); err != nil {
+		// Automatically generate a go.mod file if one doesn't already exist,
+		// because otherwise the directory cannot be used as a module. Remebering
+		// that the target might be relative
+		replDir := repl.dir
+		if !filepath.IsAbs(replDir) {
+			replDir = filepath.Join(mm.Dir, replDir)
```

Comment:
This seems a little error prone to me - I *think* that the modReplace type should probably have fields for both the absolute directory and the replacement directory; then we won't need this logic here.

Reply:
Agreed. Done